docs/src/pages/components/tabs/VerticalTabs.js

```diff
@@ -39,35 +38,24 @@ function a11yProps(index) {
   };
 }
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    flexGrow: 1,
-    backgroundColor: theme.palette.background.paper,
-    display: 'flex',
-    height: 224,
-  },
-  tabs: {
-    borderRight: `1px solid ${theme.palette.divider}`,
-  },
-}));
-
 export default function VerticalTabs() {
-  const classes = useStyles();
   const [value, setValue] = React.useState(0);
 
   const handleChange = (event, newValue) => {
     setValue(newValue);
   };
 
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{ flexGrow: 1, bgcolor: 'background.paper', display: 'flex', height: 224 }}
+    >
       <Tabs
         orientation="vertical"
         variant="scrollable"
         value={value}
         onChange={handleChange}
         aria-label="Vertical tabs example"
-        className={classes.tabs}
+        sx={{ borderRight: (theme) => `1px solid ${theme.palette.divider}` }}
```

Comment:
A different approach is used in https://next.material-ui.com/components/tabs/#basic-tabs

```jsx
<Box sx={{ borderBottom: 1, borderColor: 'divider' }}>

```

Reply:
Nice catch! Updated